### PR TITLE
Feature/Help

### PR DESCRIPTION
**Pull request by NirDafnai**
Closes: https://github.com/NirDafnai/godbot/issues/7

- Added help command
- Improved load_commands() in bot.py
- Updated skeleton

### CODE REVIEW

**Comment by torrell8** [comment]
Please use help command, closing

File: bot.py [hunk 1]
```diff
@@ -18,9 +17,12 @@ async def on_ready():
 
 def load_commands():
     logger.info("Loading commands...")
-    for file_name in os.listdir(config.COMMANDS_FOLDER):
-        if file_name.endswith(".py"):
-            bot.load_extension(f"{config.COMMANDS_FOLDER}.{file_name[:-3]}")
+    files = [f for f in os.scandir(config.COMMANDS_FOLDER) if f.is_file()]
```
Comment (benkir07) [inline bot.py:20]: Please close the file descriptor to the iterator.

File: scripts/skeleton.py [hunk 2]
```diff
@@ -1,9 +1,16 @@
 from discord.ext import commands
```
Comment (idoprog) [inline scripts/skeleton.py:1]: Please delete this file and this directory it's useless

File: cogs/help.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    @commands.Cog.listener()
+    async def on_ready(self):
+        self.logger.info("Help command ready")
+
+    @commands.command(name="help", aliases=("h",), description="Returns all commands available")
+    async def help_command(self, context: commands.Context):
+        help_message = Embed(title="Commands")
+        for command in self.bot.commands:
+            command_name = f"{self.bot.command_prefix}{command.name}"
+            if command.clean_params:
```
Comment (benkir07) [inline cogs/help.py:21]: Change to an explicit equality check to None
The variable's type is not obvious.

**Review by benkir07 — CHANGES_REQUESTED** [review]
Fix my life please.

File: scripts/skeleton.py (hunk 2 above)
Comment (NirDafnai) [reply scripts/skeleton.py:1]: No, fuck you

**Pull request merged**